data/brands/shop/variety_store.json

```diff
@@ -427,6 +427,17 @@
         "shop": "variety_store"
       }
     },
+    {
+      "displayName": "Maxi Bazar",
+      "id": "maxibazar-31d9f9",
+      "locationSet": {"include": ["fr"]},
+      "tags": {
+        "brand": "Maxi Bazar",
+        "brand:wikidata": "Q111817854",
+        "name": "Maxi Bazar",
+        "shop": "variety_store"
```

Comment:
Is this really a variety_store? look like it mainly sells furniture

Reply:
Good question.
The motto of the store is `mini prix. maxi choix` (small price, maximum choice).

There are a lot of various items that are sold in a store.
You could check their facebook page to get an impression (https://www.facebook.com/maxibazarfrance/photos/2895441030762033), or their website which would indicate various item that are being sold in the store (https://www.facebook.com/maxibazarfrance/photos/2916181412021328; https://maxibazar.fr/animalerie/; https://maxibazar.fr/poeles/; ...)

When searching on the overpass for `node["name"="Maxi Bazar"]({{bbox}});` the search reveals:
- shop=variety_store -- https://www.openstreetmap.org/node/6525598425 -- https://maxibazar.fr/antibes-olympie.html
- shop=general -- https://www.openstreetmap.org/node/6515467900 -- https://maxibazar.fr/antibes-soleau.html
- shop=supermarket -- https://www.openstreetmap.org/node/2409755581 -- https://maxibazar.fr/salon-de-provence.html
- shop=houseware -- https://www.openstreetmap.org/node/9574714018 -- https://maxibazar.fr/chambourcy.html

Given the variety of items sold in the store and their motto I would tag it as `variety_store`. But given that it's clearly not "just" a "dollar store" tagging it as `shop=interior_decoration` might fit better. Not sure. Clearly as of now it's tagged in various different ways ;-)

Reply:
This let me remember the https://github.com/osmlab/name-suggestion-index/pull/7123#discussion_r985158835

I made below [overpass query](https://overpass-turbo.eu/s/1mY7):
```overpassql
[out:csv(::id,::type,"name", "shop")][timeout:25];
nwr["name"="Maxi Bazar"];
out;
```
And here are result:

* clothes 1
* convenience 1
* department_store 1
* general 2
* gift 2
* hardware 1
* houseware 3
* **interior_decoration 8**
* supermarket 1
* **variety_store 15**
* yes 1

I guess it a furniture shop because I saw its [website](https://user-images.githubusercontent.com/42690037/197380668-27997d5e-a4f0-4b8d-9f48-ab9b2d6c48e8.png), but according to your picture, it really a variety_store.

And looks like many element was tagged as this (15/36), in #7123 we also choose the most common and major catagory.

So I have no further question about this.

Reply:
Thank you for getting this integrated and sharing your concerns!
Also thanks for verifying this with an overpass query :-)